lib/sneakers/version.rb

```diff
@@ -1,3 +1,3 @@
 module Sneakers
-  VERSION = "2.3.5"
+  VERSION = "2.3.6"
```

Comment:
Also, as a note, please don't ever do a version bump in a pull request. The
version designation is a responsibility of the release manager, whomever that
may be even if it's multiple persons or a robot.

Some projects include changelogs that include a ##next section for things on
develop that's not released that you would write the changelog notes into that
could help the release manager.

Hope this helps, cheers!

~Spaceghost